src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -302,10 +304,14 @@ export function QueryPageContainerComponent(): JSX.Element {
             setTableContext(isolateCountGroups, nrOfSelectedIsolates);
             setNrOfSelectedIsol(nrOfSelectedIsolates);
         }
+        setTableIsLoading(false)

```

Comment:
if callApiService fails or throws an error is setTableLoading never set back to 'false'?

Reply:
No it is independant of the api response status.